src/modules/addTodoTask.js

```diff
@@ -157,33 +144,9 @@ const handleEdit = (e) => {
   }
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make the commit message capitalized to make your pull request professional 👍